data/yam/autoyast/minimal_no_updates_aarch64.xml

```diff
@@ -0,0 +1,389 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE profile>
+<profile xmlns="http://www.suse.com/1.0/yast2ns" xmlns:config="http://www.suse.com/1.0/configns">
+  <add-on t="map">
+    <add_on_products t="list">
+      <listentry t="map">
+        <media_url>cd:/?devices=/dev/disk/by-id/scsi-0QEMU_QEMU_CD-ROM_cd0</media_url>
+        <product>sle-module-basesystem</product>
+        <product_dir>/Module-Basesystem</product_dir>
+      </listentry>
+      <listentry t="map">
+        <media_url>cd:/?devices=/dev/disk/by-id/scsi-0QEMU_QEMU_CD-ROM_cd0</media_url>
+        <product>sle-module-server-applications</product>
+        <product_dir>/Module-Server-Applications</product_dir>
+      </listentry>
+    </add_on_products>
+  </add-on>
+  <bootloader t="map">
+    <global t="map">
+      <append>splash=silent plymouth.ignore-serial-consoles console=ttyAMA0 console=tty YUI_HTTP_PORT=39125 YUI_HTTP_REMOTE=1 YUI_REUSE_PORT=1 kernel.softlockup_panic=1 quiet security= mitigations=auto</append>
```

Comment:
the port is dynamic, have you checked other AutoYaST files? I find here a lot of information not really needed. AutoYaST just woud put it there for you.

Reply:
actually if you can do the zypper ref/zypper ar/zypper dup with a system unregistered (this is how you did I think according to your ay profiles) it should be fine. The only different is that if the system contains the regcode or not, otherwise it should install the same because there are not update channels.

Reply:
I'm now trying with a stripped down "mini-like" version to see if works. We can use autoyast without registration if we are using the Full medium. We can't with the Online one.